kpi/views/browser_kpis.view.lkml

```diff
@@ -0,0 +1,44 @@
+include: "./unified_metrics.view.lkml"
+
+view: browser_kpis {
+  extends: [unified_metrics]
```

Comment:
Extending the view instead of the explore also updates this view when `unified_metrics` is updated (e.g. when new fields are being added)